### PR TITLE
docs: added getResponse instance method

### LINKED ISSUE #185: add info about getResponse to docs

### PR DESCRIPTION
**Pull request by AndyOGo**
fixes #185 

**Changes:**

- ~~add `getResponse` instance method to readme~~ already done

- adds `getResponse` example which just `alert`s it

### CODE REVIEW

File: example/src/modules/example.js [hunk 1]
```diff
@@ -78,6 +78,14 @@ export default class Example extends Component<Props, State> {
     }
   };
 
+  getResponseRecaptcha = () => {
+    if (this.captcha) {
+      this.captcha.getResponse().then(response => {
+        alert(response);
+      });
```
Comment (AndyOGo) [inline example/src/modules/example.js]: @sarneeh 
The google `getResponse` API will return just an `""` empty string until the challenge is resolved.

I'm just curious if it should be treated as a Promise rejection 🤔 
Current implementation would only reject the Promise if the underlying Google API would `throw`, which I consider valid behaviour too.

**Comment by sarneeh** [comment]
Hey @AndyOGo, I guess you didn't notice, that I've already created a PR for this: #187 :) We can keep both tho: we'll just keep the example from this PR.

**Comment by AndyOGo** [comment]
@sarneeh Thanks for your quick answer.
Sorry I didn't check for open PR's.
Feel free to cherry pick or close this one

**Comment by AndyOGo** [comment]
@sarneeh 
Do you have any time soon, to check if you agree with this example?

**Pull request merged**